subsys/bluetooth/controller/hci/hci.c

```diff
@@ -5666,7 +5666,7 @@ int hci_iso_handle(struct net_buf *buf, struct net_buf **evt)
 
 	iso_hdr = net_buf_pull_mem(buf, sizeof(*iso_hdr));
 	handle = sys_le16_to_cpu(iso_hdr->handle);
-	len = sys_le16_to_cpu(iso_hdr->len);
+	len = bt_iso_hdr_len(sys_le16_to_cpu(iso_hdr->len));
```

Comment:
Should this have been
```suggestion
	len = sys_le16_to_cpu(bt_iso_hdr_len(iso_hdr->len));
```

As done in lines 5709 and 5710?

Reply:
I can see how that looks a bit inconsistant. In case of `iso_hdr->len`, this is a simple case of truncating the size of the number, and the existing macro is designed for "cpu-domain" masking. For this reason, we must convert the value to cpu domain before applying the mask.
For `iso_data_hdr->slen`, the macro is designed directly from the spec in network domain, and must be used before converting to cpu domain.